.github/workflows/release-trigger-builds.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: release-trigger-builds
+
+on:
+  workflow_dispatch:
+    inputs:
+      dispatch_target_release_branch:
+        description: "Cloud release branch to trigger"
+        required: true
+      dispatch_content:
+        description: "Regular (only centreon named components) or Full (every component, including php and extra libs)"
+        required: true
+        type: choice
+        options:
+          - REGULAR
+          - FULL
+
+
+jobs:
+  release-trigger-builds:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Github CLI
+        run: |
+          echo "Installing GH CLI."
+          if ! command -v gh &> /dev/null; then
+            echo "Installing GH CLI."
+            type -p curl >/dev/null || (sudo apt-get update && sudo apt-get install curl -y)
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+            sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+            sudo apt-get update
+            sudo apt-get install gh -y
+          else
+            echo "GH CLI is already installed."
+          fi
+        shell: bash
+
+      - name: Trigger selected branches
+        run: |
+          set -eux
+
+          # VARS
+          # names defined in workflow files per components
+          COMPONENTS_OSS=("awie" "dsm" "gorgone" "ha" "open-tickets" "web")
+          COMPONENTS_OSS_FULL=("awie" "dsm" "gorgone" "ha" "open-tickets" "web")
```

Comment:
indeed they are, I just leave it here to avoid too much divergence between the logic (should this cript be backported to older branches, it will have a real _FULL mode with extra components (such as widgets)